l1-contracts/gas_benchmark.md

```diff
@@ -1,20 +1,20 @@
 Function                 | Metric  | No Validators (gas/tx) | 100 Validators (gas/tx) |         Δ Gas (gas/tx) |   % Overhead
 -------------------------+---------+------------------------+-------------------------+------------------------+-----------------
-forward                  | Min     |     298278 (828.55)    |      576701 (1601.95)   |     278423 (773.40)    |      93.34%
-forward                  | Avg     |     310153 (861.54)    |      588784 (1635.51)   |     278631 (773.98)    |      89.84%
-forward                  | Median  |     309029 (858.41)    |      587794 (1632.76)   |     278765 (774.35)    |      90.21%
-forward                  | Max     |     321743 (893.73)    |      600650 (1668.47)   |     278907 (774.74)    |      86.69%
-forward                  | # Calls |                    100 |                     100 |                      0 |       0.00%
+propose                  | Min     |     278948 (774.86)    |      545790 (1516.08)   |     266842 (741.23)    |      95.66%
```

Comment:
not sure what's going on here..

Reply:
Just an annoying result of l1-contracts storing benchmark results in repo